src/components/markdown/references/references.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { useContext } from 'react'
```

Comment:
It seems like prettier is complaining about the format on this file.

You can run `yarn format` to fix it and then make a commit of the changes.